modules/xfem/doc/content/modules/xfem/index.md

```diff
@@ -1,6 +1,61 @@
 # XFEM Module
 
-!alert construction
-The XFEM module needs some documentation.
+!row!
+!col! small=12 medium=6 large=9
+The XFEM module implements the extended finite element method (XFEM) in the MOOSE framework. It can
+be used in conjuction with any number of other physics modules to enrich the solutions to partial
+differential equations (PDEs) with some form of discontinuity. XFEM theory and algorithms employed
+throughout this module are detailed in the following links:
 
-!syntax complete groups=XFEMApp
+- [Theory Manual](xfem/theory/theory.md)
```

Comment:
I think the difference between them is sort of general overview of the methods involved in our implementation (theory) versus specific techniques involved in implementing the methods (algorithms), but the distinction here is just what I thought based on what @jiangwen84 said he wanted to do for his documentation story ("Are you working on the index.md? Besides some cool pictures, Ben was saying we should add some code algorithm descriptions there." ... "Maybe I could use my 1 day to work on the algorithms part.") I envisioned the theory section being pretty heavy on equations, highlighting the differences between "classic" XFEM and the phantom node method's matrix implementation (and consequently why we need to enforce continuity of solutions/fluxes at interfaces). The theory section I was thinking would have sections such as quadrature weighting, phantom node method, variable enforcement constraints, and near tip enrichment (this would probably need to be done by Wen) whereas I was thinking the algorithms section would be more specific examples of techniques that are implemented to accomplish what we spelled out in the theory section. Does this make sense?

Reply:
@james-tompkins I think we can call our implementation as code manual, instead of theory. I agree with Ben, theory and algorithm might refer to a same thing.

Reply:
Okay, I'll replace the 'theory' and 'algorithms' section with one 'theory' section.